server/start.go

```diff
@@ -229,12 +228,14 @@ func startStandAlone(ctx *Context, appCreator types.AppCreator) error {
 
 	err = svr.Start()
 	if err != nil {
-		tmos.Exit(err.Error())
+		fmt.Printf(err.Error() + "\n")
```

Comment:
Why don't we just use Println?

Reply:
I just blindly accepted what was on Tendermint 🤦 There's an improvement to be made over there then lol